src/main/riddl/AllIdsProjections/allIdsProjections.riddl

```diff
@@ -0,0 +1,103 @@
+query GetAllTenantIds is {}
+result AllTenantIds is {tenants: TenantId*}
+
+query GetAllOrgIds is {}
+result AllOrgIds is {orgs: OrganizationId*}
+
+query GetAllEventIds is {}
+result AllEventIds {events: EventId*}
+
+query GetAllMemberIds is {}
+result AllMemberIds is {members: MemberId*}
+
+query GetAllStoreIds is {}
+result AllStoreIds is {stores: StoreId*}
+
+query GetAllSkus is {}
+result AllSkus is {skus: SKU*}
+
+context AllIdsProjections is  {
+  projection AllTenantIdsView is {
+     fields {
+        tenant: TenantId
+    }
+    handler AllTenantIdsViewHandler is {
+        on event TenantEstablished {
+            then "create row with tenant set to TenantEstablished.tenantId"
+        }
+        on query GetAllTenantIds {
+            then "reply with AllTenantIds packed with entire table"
```

Comment:
I am not looking at type definitions ATM, but I wonder if it would be worth qualifying this with all "Active" TenantIDs?